web/testing/prediction-ui/index.html

```diff
@@ -41,6 +41,7 @@
 		    attachType:'auto'
       }).then(function() {
         kmw.addKeyboards({id:'us',name:'English',languages:{id:'en',name:'English'}, filename:'../us-1.0.js'});
+        kmw.addKeyboards({id:'lao_2008_basic',name:'Lao',languages:{id:'lo',name:'Lao'}, filename:'../lao_2008_basic.js'});
```

Comment:
btw, the en wordlist model is only getting registered on initial page load.

Reply:
Yep, that's correct.  'Native' Web only needs to register a lexical model once; that's why I chose the word `register`.  Any registered lexical model will be used for any keyboard with matching language tag until it is either replaced or deregistered.

Reply:
PEBKAC - I had only rebuilt KMW embedded for testing in the app.